Forge.TreeWalker/src/TreeWalkerSession.cs

```diff
@@ -270,6 +275,15 @@ public async Task<string> GetLastTreeAction()
             }
         }
 
+        /// <summary>
+        /// Get the string context if the actions in this TreeNodes are skipped.
+        /// Usually, when there is no action skipped, the return value is null.
+        /// </summary>
+        public string GetCurrentNodeSkipActionContext()
```

Comment:
>GetCurrentNodeSkipActionContext [](start = 22, length = 31)

Update summary to match interface summary.

Reply:
updated, also changed "TreeNode" to "tree node" for consistency. 
and updated the comment for private string currentNodeSkipActionContext;

---
In reply to: [505915013](https://github.com/microsoft/Forge/pull/32#discussion_r505915013) [](ancestors = 505915013)